.taskcluster.yml

```diff
@@ -1,245 +1,118 @@
-      tasks_priority: highest
+    $if: 'tasks_for in ["github-pull-request"]'
+    then:
+      $let: {}
+      in:
+        taskId: '${as_slugid("decision")}'
+        taskGroupId: '${as_slugid("decision")}'
+        schedulerId: 'mobile-level-1'
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 day'}
+        expires: {$fromNow: '1 year 1 second'}  # 1 second so artifacts expire first, despite rounding errors
+        provisionerId: "aws-provisioner-v1"
+        workerType: "mobile-1-decision"
+        scopes:
+          - 'assume:repo:github.com/${event.pull_request.base.repo.full_name}:pull-request'
+        metadata:
+          name: Decision Task
+          description: Decision Task
+          owner: '${owner_email}'
+          source: '${event.pull_request.head.repo.html_url}/raw/${head_sha}/.taskcluster.yml'
 
-      short_head_branch:
-        $if: 'head_branch[:10] == "refs/tags/"'
-        then: {$eval: 'head_branch[10:]'}
-        else:
-          $if: 'head_branch[:11] == "refs/heads/"'
-          then: {$eval: 'head_branch[11:]'}
-          else: ${head_branch}
+        requires: all-completed
+        routes: [checks]
+        priority: lowest
+        retries: 5
 
-      assume_scope_prefix: assume:repo:github.com/${github_repository_full_name}
-    in:
-      $let:
-        default_task_definition:
-          taskId: ${decision_task_id}
-          taskGroupId: ${decision_task_id}  # Must be explicit because of Chain of Trust
-          schedulerId: ${scheduler_id}
-          created: {$fromNow: ''}
-          deadline: {$fromNow: '2 hours'}
-          expires: ${expires_in}
-          provisionerId: aws-provisioner-v1
-          workerType: mobile-${trust_level}-decision
-          priority: ${tasks_priority}
-          requires: all-completed   # Must be explicit because of Chain of Trust
-          retries: 5
-          routes:
-            $flatten:
-              - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
-              - $if: 'trust_level == 3'
+        payload:
+          maxRunTime: 3600
+          env:
+            # run-task uses these to check out the source; the inputs
+            # to `mach taskgraph decision` are all on the command line.
+            $merge:
+              - MOBILE_BASE_REPOSITORY: '${event.pull_request.base.repo.html_url}'
+                MOBILE_HEAD_REPOSITORY: '${event.pull_request.head.repo.html_url}'
+                MOBILE_HEAD_REF: '${event.pull_request.head.ref}'
+                MOBILE_HEAD_REV: '${event.pull_request.head.sha}'
+                MOBILE_REPOSITORY_TYPE: git
+                REPOSITORIES: {$json: {mobile: "Reference Browser"}}
+                ANDROID_SDK_ROOT: /builds/worker/checkouts/android-sdk
+                ANDROID_HOME: /builds/worker/checkouts/android-sdk
+                MOZ_FETCHES_DIR: /builds/worker/checkouts/fetches
+                MOZ_FETCHES:
```

Comment:
It is a uniform way for tasks to pull artifacts from other tasks (see [here](https://searchfox.org/mozilla-central/rev/3366c3d24f1c3818df37ec0818833bf085e41a53/taskcluster/taskgraph/transforms/job/__init__.py#159-250), and implemented [here](https://searchfox.org/mozilla-central/rev/3366c3d24f1c3818df37ec0818833bf085e41a53/taskcluster/scripts/misc/fetch-content#582-602)). I (as a hack that I have since unwound) added a `url` method to get arbitrary URL instead. 
---
I've since moved the android-sdk handling into a script for use in the decision and other tasks. (See [here](https://mozilla.slack.com/archives/GHVMPRM3Q/p1565730560180000) for part of the reason).